modules/dag-shared/src/main/scala/org/tessellation/dag/snapshot/ActiveTip.scala

```diff
@@ -1,15 +1,20 @@
 package org.tessellation.dag.snapshot
 
 import org.tessellation.dag.domain.block.BlockReference
+import org.tessellation.ext.cats.data.OrderBasedOrdering
 
-import derevo.cats.{eqv, show}
+import derevo.cats.{order, show}
 import derevo.derive
 import eu.timepit.refined.cats._
 import eu.timepit.refined.types.numeric.NonNegLong
 
-@derive(eqv, show)
+@derive(order, show)
 case class ActiveTip(
   block: BlockReference,
   usageCount: NonNegLong,
   introducedAt: SnapshotOrdinal
 )
+
+object ActiveTip {
+  implicit object OrderingInstance extends OrderBasedOrdering[ActiveTip]
```

Comment:
It's called `Ordering` in `BlockAsActiveTip` maybe we can use the same naming convention everywhere?

Reply:
True, I think that `OrderingInstance` is a better name, because it doesn't conflict with `scala.math.Ordering`. I've noticed the inconsistency and I've put a comment about it https://github.com/Constellation-Labs/tessellation/pull/262#discussion_r887828217